datasets/bagh/bagh.json

```diff
@@ -5,6 +5,13 @@
   "status": "beschikbaar",
   "version": "0.0.1",
   "crs": "EPSG:28992",
+  "identifier": "identificatie",
+  "temporal": {
+    "identifier": "volgnummer",
+    "dimensions": {
+      "geldigOp": ["begin geldigheid", "eind geldigheid"]
```

Comment:
`"geldigOp": ["beginGeldigheid", "eindGeldigheid"]`, zo is het in andere datasets.

Reply:
It was correct back in June :D 
This PR was kind of reference for the rest of temporal datasets. Closing.